Documentation/rkt-vs-other-projects.md

```diff
@@ -14,7 +14,8 @@ This document describes how rkt compares to various other projects in the contai
 
 ## rkt vs Docker
 
-The Docker Engine, or simply Docker, is an application container runtime system that runs as a centralized API daemon. Docker can take a ["Docker Image"](https://github.com/docker/docker/blob/master/image/spec/v1.md) name, such as `quay.io/coreos/etcd`, and download, execute, and monitor the application.
+The Docker Engine, or simply Docker, is an application container runtime system that runs as a centralized API daemon.
```

Comment:
The Docker Engine is an application container runtime implemented as a central monolithic API daemon.